Lets Do This/Views/Campaign/LDTCampaignDetailCampaignCell.m

```diff
@@ -95,11 +107,35 @@ - (void)setCoverImageURL:(NSURL *)coverImageURL {
     }];
 }
 
-- (void)setDisplaySubmitReportbackButton:(BOOL)displaySubmitReportbackButton {
-    _displaySubmitReportbackButton = displaySubmitReportbackButton;
-    if (displaySubmitReportbackButton) {
-        // @todo: Create public SubmitReportbackButtonTitle property.
-        [self.submitReportbackButton setTitle:@"Prove it".uppercaseString forState:UIControlStateNormal];
+- (void)setDisplayCampaignDetailsView:(BOOL)displayCampaignDetailsView {
+    _displayCampaignDetailsView = displayCampaignDetailsView;
+    if (!_displayCampaignDetailsView) {
+        self.diagonalShapeLayer.hidden = YES;
+        // Hack to avoid UILabels preventing  campaignDetailsView from fully collapsing.
```

Comment:
Also not having luck by setting a height constraint of 0 on the `UILabel` and setting its priority to 999, the label still appears.